### PR TITLE
Adding addon relations

### PR DESCRIPTION
**Pull request by OlegYef**

### CODE REVIEW

File: endpoints/addons.ts [hunk 1]
```diff
@@ -129,6 +137,15 @@ export class AddonEndpoint extends Endpoint<Addon> {
...
     // data = new AddonDataEndpoint(this.service);
+
+    sencodeQueryParams(params: any) {
+        const ret: string[] = [];
+        Object.keys(params).forEach((key) => {
+            ret.push(key + '=' + encodeURIComponent(params[key]));
+        });
+        return ret.join('&');
+    }
+
```
Comment (Chasky-H) [inline endpoints/addons.ts]: what is this???

File: entities/addons.ts [hunk 2]
```diff
@@ -64,3 +64,18 @@ export interface AddonDataScheme {
     };
     Validator?: string;
 }
+
+type RelationType = 'AddonAPI' | 'NgComponent' | 'Navigation';
```
Comment (Chasky-H) [inline entities/addons.ts:68]: Please export this

**Review by Chasky-H — CHANGES_REQUESTED** [review]

File: endpoints/addons.ts (hunk 1 above)
Comment (OrenVildermanPepperi) [reply endpoints/addons.ts]: This was removed, you are not looking on the correct version

File: entities/addons.ts (hunk 2 above)
Comment (OrenVildermanPepperi) [reply entities/addons.ts]: Exported

**Review by Chasky-H — APPROVED** [review]

**Pull request merged**